### PR TITLE
new DATA sample

### PR DESCRIPTION
**Pull request by dimaspirit**
**Made/Proposed changes:**

- Data sample;

**How should this be manually tested?**
`npm run start` & localhost:8080/samples/data

**Does the documentation need an update?**
No

### CODE REVIEW

File: samples/data/js/app.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 /* eslint no-alert: "off" */
 /* eslint no-console: "off" */
-/* global QB_CREDS:true, QB_CONFIG:true, User:true, Places:true, Map:true, Handlebars:true */
+/* global QB_CREDS:true, QB_CONFIG:true, User:true, Places:true, Checkin:true, WMap:true, Handlebars:true */
+
 
 function App() {
   this.ui = {
```
Comment (Vladlukhanin) [inline samples/data/js/app.js]: semicolon

File: samples/webrtc/styles.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,800&subset=latin,latin-ext);
+/*@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,800&subset=latin,latin-ext);*/
```
Comment (Vladlukhanin) [inline samples/webrtc/styles.css]: ?

**Review by Vladlukhanin — COMMENTED** [review]
1) Don't forget about multiple requests to get all data (for example - Places.sync(), Checkin.get() )
2) App can't detect current position after allowing permissions in Safari browser.

**Comment by soulfly** [comment]
Approved

**Pull request merged**